### DISCUSSION TITLE
Is it possible to nest `expand`?

### DISCUSSION THREAD

**Discussion by g1sbi** [pocketbase/pocketbase#2805] [Q&A] [answered]
I have a user, which has some threads saved in their account. These threads have a relation to a bunch of updates. So far, I have this that fetches user and threads:
```ts
  const user = await pb.collection("users").getOne(id, {
    expand: "followed",
  });

```
How do I expand to the updates inside the threads?

 I have read in the "Expanding Relations" part of the docs: `Nested relations are supported via dot-notation and up to 6-levels depth.` , however I can't figure out how to do it.

**Comment by ganigeorgiev** [comment, marked as answer]
Here is an example:

```
expand: "followed.someRelField"
```
In the above `someRelField` is a relation field inside the collection that is referenced by `followed`.

If the above doesn't help, please provide more context about your collections structure (_you can also export your collections configuration as json from the Admin UI > Settings > Export collections_).
